SPEC.md

```diff
@@ -133,14 +133,11 @@ make it act as plain whitespace, even if it spreads across multiple lines.
 
 #### Equals Sign
 
-Any of the following characters may be used as equals signs in properties:
+Only the following character may be used as equals signs in properties:
 
 | Name | Character | Code Point |
 |----|-----|----|
 | EQUALS SIGN | `=` | `U+003D` |
-| SMALL EQUALS SIGN | `﹦` | `U+FE66` |
-| FULLWIDTH EQUALS SIGN | `＝` | `U+FF1D` |
-| HEAVY EQUALS SIGN | `🟰` | `U+1F7F0` |
```

Comment:
Is it necessary to keep this section? It's has only one reference, in the previous section, that could be replaced with "equals sign (`=`, code point `U+003D`)". Regardless, the previous section might need to be reworded anyway (#401).

Reply:
done